components-rs/Cargo.toml

```diff
@@ -8,7 +8,7 @@ crate-type = ["staticlib", "cdylib"]
 path = "lib.rs"
 
 [dependencies]
-ddcommon = { path = "../libdatadog/ddcommon" }
+ddcommon = { path = "../libdatadog/ddcommon", features = ["cgroup_testing"] }
```

Comment:
If this is for tests, you probably don't want this feature as a regular dependency. You can leave the ddcommon dependency as it was here, and under dev-dependencies add it again with this feature.